MIGRATION.md

```diff
@@ -41,6 +42,19 @@ These guides below are provided to ease the transition of existing applications
 * [Onfido iOS SDK 3.0.0 Migration Guide](#onfido-sdk-300-migration-guide)
 
 
+## Onfido iOS SDK 18.8.0 Migration Guide
+
+### String Changes
+
+#### Changed
+
+⚠️ Most localisation keys now renamed. Use [migrate-keys.rb](keys_migration/migrate-keys.rb) script and key mapping file [key_migration_18_7_0_mapping.json](keys_migration/key_migration_18_7_0_mapping.json) to migrate from 18.7.0 to 18.8.0. To use run following command:
```

Comment:
the `keys_migration/migrate-keys.rb` and `key_migration_18_7_0_mapping.json` not exist in the pr